cypress/e2e/index.cy.js

```diff
@@ -0,0 +1,15 @@
+describe('Home', () => {
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  it('should render a heading', () => {
+    cy.get('h1').should('exist');
+  });
+
+  it('renders multiple component cards', () => {
+    cy.get('[data-cy=component-card]').each((card) => {
```

Comment:
Thanks for 1) Moving these tests to Cypress, and 2) changing this to use a `data-cy` attribute instead of the `listitem` role. Much more future-proof!